Extensions/3D/Model3DRuntimeObject3DRenderer.ts

```diff
@@ -334,7 +334,8 @@ namespace gdjs {
       if (!this._action) {
         return true;
       }
-      return !this._action.isRunning();
+      //If this animation is looping, we dont want it to be able to send "true" as it must be endless.
```

Comment:
nitpick, Always add a space between `//` and the start of your text.
It's a convention we have in the codebase to ease readability.
You can do it on your comments below too